docs/learn/architecture/hubs.md

```diff
@@ -58,4 +58,4 @@ You may need a Hub if you are building an app that wants to read or write Farcas
 
 **2. Are there rewards for running a Hub?**
 
-Farcaster does not provide rewards for Hubs and does not plan to. Warpcast, a company building on Farcaster, gives Hub runners minor rewards but may change this in the future.
+Farcaster does not provide rewards for Hubs and does not plan to. Warpcast, a company building on Farcaster, has stopped providing small rewards for running a Hub.
```

Comment:
suggestion: I think we should remove the sentence, to avoid implying resumption of rewards in the future
```suggestion
Farcaster does not provide rewards for Hubs and does not plan to.
```